CAN-D/Drivers/BSP/Inc/stm32302c_custom_gps.h

```diff
@@ -0,0 +1,53 @@
+/**
+  ******************************************************************************
+  * @file    stm32302c_custom_gps.h
+  * @author  Mark Griffith
+  * @brief   This file contains the common defines and functions prototypes for
+  *          the stm32303c_custom_gps.c driver.
+  ******************************************************************************
+  */
+
+/* Define to prevent recursive inclusion -------------------------------------*/
+#ifndef __STM32303C_CUSTOM_GPS_H
+#define __STM32303C_CUSTOM_GPS_H
+
+#ifdef __cplusplus
+ extern "C" {
+#endif
+
+
+#include "stm32303c_custom.h"
+
+/* Commands to set the update rate from once a second (1 Hz) to 10 times a second (10Hz) */
+// Note that these only control the rate at which the position is echoed, to actually speed up the
+// position fix you must also send one of the position fix rate commands below too.
+#define PMTK_SET_NMEA_UPDATE_100_MILLIHERTZ	"$PMTK220,10000*2F\r\n" /* Once every 10 seconds, 100 millihertz. */
```

Comment:
I agree. This was just because my editor switched to using hardtops of size 4 randomly. Probably after copying some code from stackoverflow. I think I've fixed this.

Reply:
It's annoying because I only see the spacing mismatch when it's displayed on GitHub, so I didn't know it was mismatched.